examples/cli/cli.go

```diff
@@ -0,0 +1,39 @@
+package cli
+
+import (
+	"context"
+	"flag"
+	"github.com/zmb3/spotify/v2"
+	spotifyauth "github.com/zmb3/spotify/v2/auth"
+	"golang.org/x/oauth2"
+	"log"
+)
+
+var auth = spotifyauth.New(spotifyauth.WithRedirectURL("http://localhost:3000/login_check"))
```

Comment:
Do we need to set a redirect URL here ?

Reply:
Yes, it is required by Spotify when exchanging the authorization code by the token for validation purposes (https://developer.spotify.com/documentation/general/guides/authorization/code-flow#request-access-token)